App.tsx

```diff
@@ -77,50 +20,25 @@ const App = () => {
     return <SplashScreen />;
   }
   return (
-    <SafeAreaView style={backgroundStyle}>
-      <StatusBar barStyle={isDarkMode ? 'light-content' : 'dark-content'} />
-      <ScrollView contentInsetAdjustmentBehavior="automatic" style={backgroundStyle}>
-        <Header />
-        <View
-          style={{
-            backgroundColor: isDarkMode ? Colors.black : Colors.white,
+    <NavigationContainer>
+      <Stack.Navigator>
+        <Stack.Screen
+          name="Home"
+          component={HomeScreen}
+          options={{
+            header: () => <Header title="HomeScreen" hasHelpButton />,
```

Comment:
Misschien een idee om die titles ook in een const file te gooien?

Reply:
Deze titles zijn nu alleen voor het testen. Elk scherm (behalve het hoofd- en helpscherm) heeft straks een dynamische title toch?